forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -349,6 +357,21 @@ void SADSRG::fill_Fdiag(BlockedTensor& F, std::vector<double>& Fa) {
     });
 }
 
+void SADSRG::build_custom_fock() {
+    size_t ncmo = mo_space_info_->size("CORRELATED");
```

Comment:
This "custom" Fock does not seem very customizable. It's just the Fock matrix using the ref. 1RDM. Which is a fine thing to do, but the function name does not communicate that.